rs-e2e/src/main/java/gov/hhs/cdc/trustedintermediary/rse2e/AzureBlobHelper.java

```diff
@@ -0,0 +1,22 @@
+package gov.hhs.cdc.trustedintermediary.rse2e;
+
+import java.time.LocalDate;
+
+public class AzureBlobHelper {
```

Comment:
The other files with classes e.g 'AzureBlobFileFetch', 'LocalFileFetcher' have comments above the class. Do the new classes 'AzureBlobHelper' and 'AzureBlobOrganizer' need them too?

Reply:
I'll add that. Thanks for reviewing!